src/tests/functional/base_func_tests/include/behavior/compiled_model/compiled_model_base.hpp

```diff
@@ -764,6 +764,39 @@ TEST_P(CompiledModelSetType, canSetInputOutputTypeAndCompileModel) {
     OV_ASSERT_NO_THROW(core.compile_model(model, target_device, configuration));
 }
 
+TEST_P(OVCompiledModelBaseTest, import_from_istream) {
+    if (0 != target_device.compare(utils::DEVICE_CPU) &&
```

Comment:
Check for skipped test if the file '`skip_test_config.cpp` is created for device and put skip pattern there.
The NPU plugin may use some `.xml` file like:
`src/plugins/intel_npu/tests/functional/shared_tests_instances/npu_skip_func_tests.xml`

Reply:
done